src/main/kotlin/Properties/Tamir/PropertiesTamirClassEx.kt

```diff
@@ -0,0 +1,37 @@
+package Properties.Tamir
+import java.time.DayOfWeek
+
+//1. create class called Store that initlize by day and list of products
+data class Store(val day: DayOfWeek,val products: List<Product>)
+{
+    //2. add property to that indicate if the store is open the store is open all the day expect saturday
+    val shomeretShabat:Boolean get() {
```

Comment:
space after :, its not critical but its important to keep on code style

Reply:
Fix it right now
Thanks for the review Tamir!